CHANGELOG.md

```diff
@@ -37,7 +37,7 @@
 * [BUGFIX] Cortex: Fix GRPC stream clients not honoring overrides for call options. #5797
 * [BUGFIX] Ring DDB: Fix lifecycle for ring counting unhealthy pods as healthy. #5838
 * [BUGFIX] Ring DDB: Fix region assignment. #5842
-
+* [BUGFIX] Improve GetRules performance #5805
```

Comment:
The description doesn't sound like a bug. If it is a bug fix rather than an improvement, please mention what bug it is trying to fix like other changelog items do.
And `Improve GetRules performance` is very vague, please be explicit. It would be better to provide some data points like Go benchmark tests.